resources/assets/js/fakeData/fakeManager.ts

```diff
@@ -22,7 +22,7 @@ export const fakeManager = (id = 1, user_id = 1): Manager => ({
     learning_path:
       "I did Physics back in school, but honestly I find learning on the job way more fun than academia. These days, my main learning path is to work with people who are better than me and absorb as much as I can. I also do a fair share of throwing myself at new problems I know little about. It often leads to uncomfortable situations… but is great for learning.",
     about_me:
-      "I like games of pretty much any short, including sports, video games and table-top. In my free time I listen to podcasts about electric cars, science and skepticism. I'm also very lucky to share a home with my wife and son (soon to add a second!)",
+      "I like games of pretty much any short, including sports, video games and table-top. In my free time I listen to podcasts about electric cars, science and scepticism. I'm also very lucky to share a home with my wife and son (soon to add a second!)",
```

Comment:
Not sure how I feel about this one. I've always seen it spelled skeptic, and most articles point to skeptic as the Canadian/American spelling, but sceptic as the British so I guess we could go either way with our colony heritage?